packages/language-server/src/__test__/multifile/MultifileHelper.ts

```diff
@@ -0,0 +1,83 @@
+import { URI } from 'vscode-uri'
+import { PrismaSchema, SchemaDocument } from '../../lib/Schema'
+import path from 'path'
+import { getMultifileSchema } from '../helper'
+import { Position, TextEdit } from 'vscode-languageserver'
+import { TextDocument } from 'vscode-languageserver-textdocument'
+
+export async function getMultifileHelper(fixturePath: string) {
+  const schema = await getMultifileSchema(fixturePath)
+  const helper = new MultfileHelper(`/multifile/${fixturePath}`, schema)
+  return helper
+}
+
+class MultfileHelper {
+  constructor(
+    private baseDir: string,
+    readonly schema: PrismaSchema,
+  ) {}
```

Comment:
I don't think I've seen this syntax b4, is this a shorthand? i.e. this is equivalent?

```ts
private baseDir: string;
readonly schema: PrismaSchema;

constructor(baseDir: string, schema: PrismaSchema) {
  this.baseDir = baseDir
  this.schema = schema
}
```